pygmt/tests/test_blockm.py

```diff
@@ -12,7 +12,7 @@
 from pygmt import blockmean, blockmode
 from pygmt.datasets import load_sample_data
 from pygmt.exceptions import GMTInvalidInput
-from pygmt.helpers import GMTTempFile, data_kind
```

Comment:
These `assert`s are already covered by the `data_kind` doctests and can be removed.